apps/src/code-studio/pd/workshop_dashboard/components/session_form_part.jsx

```diff
@@ -79,30 +86,60 @@ var SessionFormPart = React.createClass({
     return null;
   },
 
+  // Return the time 1 minute before specified maxTime, bounded by MAX_TIME
+  getBoundedMaxTime(maxTime) {
+    let m = moment(maxTime, TIME_FORMAT);
```

Comment:
Might be worth going through all of your usages of `let` and seeing if you can change them to const. In most cases I think you will be able to.